exonum-java-binding/core/rust/src/handle/resource_manager/imp.rs

```diff
@@ -129,25 +129,6 @@ pub fn remove_handle<T: 'static>(handle: Handle) {
     remove_handle_impl::<T>(handle, HandleOwnershipType::JavaOwned);
 }
 
-/// Adds native-owned handle to the resource manager.
-///
-/// # Panics
-///
-/// See `add_handle_impl` for the details.
-pub fn register_handle<T: 'static>(handle: Handle) {
-    assert_ne!(handle, 0);
-    add_handle_impl::<T>(handle, HandleOwnershipType::NativeOwned);
```

Comment:
NativeOwned — is this ownership type used anywhere? Shan't we revert all changes from 8c395c571b1a9097c217a2920b010fd466865b8b?

Reply:
Thanks, removed all that stuff